src/views/modules/map.php

```diff
@@ -16,4 +16,4 @@
 
 $style = apply_filters( 'tribe_events_embedded_map_style', "height: $height; width: $width", $index );
 ?>
-<div id="tribe-events-gmap-<?php esc_attr_e( $index ) ?>" style="<?php esc_attr_e( $style ) ?>"></div><!-- #tribe-events-gmap-<?php esc_attr_e( $index ) ?> -->
+<div id="tribe-events-gmap-<?php esc_attr( $index ) ?>" style="<?php esc_attr_e( $style ) ?>"></div><!-- #tribe-events-gmap-<?php esc_attr( $index ) ?> -->
```

Comment:
This will not display the index as you forgot to call echo in the php tag `<?php echo esc_attr( $index ) ?>`.

Reply:
We shouldn't use _e() when it's not necessary. It can't be officially translated when passed a variable for the translation string.

Just replace these with `<?php echo esc_attr( $index ); ?>` and we can call this golden.